CMakeLists.txt

```diff
@@ -17,7 +17,7 @@ endif()
 
 enable_language(C)
 
-set(CMAKE_CXX_STANDARD 11)
+set(CMAKE_CXX_STANDARD 14)
```

Comment:
You really need C++14 here?

Reply:
I don't believe so. I was just able to test it with C++11 just fine. I think I might have accidentally introduced this into the ExaCMech build system as well. I can't remember why though, but I should fix that there as well...